packages/bpk-component-rating/src/BpkRating-test.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import BpkRating from '../index';
+
+describe('BpkRating', () => {
+  it('should render correctly', () => {
+    const tree = renderer
+      .create(
+        <BpkRating
+          ariaLabel="6.7 Average might recommend"
+          title="Average"
+          subtitle="Might recommend"
+          value={6.7}
+        />,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
We should have some more tests here, I'm thinking:

- One for each type of rating (low, medium, high) to test the logic of setting the right class depending on the value.
- A couple of 'edge case' ones - for example what happens if you put `-1`, or `15`, or `4.1964` to test that the value is correctly sanitised.

Reply:
Ah yes good catch! Was meant to do that 😄